frontend/packages/dev-console/integration-tests/package.json

```diff
@@ -8,6 +8,10 @@
   },
   "scripts": {
     "test-cypress": "../../../node_modules/.bin/cypress open --env openshift=true",
-    "test-cypress-headless": "node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/project-creation.feature\";"
+    "clean-reports": "rm -rf ../../../gui_test_screenshots",
+    "cypress-merge": "mochawesome-merge ../../../gui_test_screenshots/cypress_report*.json > ../../../gui_test_screenshots/cypress.json",
+    "cypress-generate": "marge -o ../../../gui_test_screenshots/ -f cypress-report -t 'OpenShift DevConsole Cypress Test Results' -p 'OpenShift Cypress Test Results' --showPassed false --assetsDir ../../../gui_test_screenshots/cypress/assets ../../../gui_test_screenshots/cypress.json",
+    "test-headless": "node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/project-creation.feature\";",
+    "test-cypress-headless": "yarn run clean-reports && yarn run test-headless && yarn run cypress-merge && yarn run cypress-generate"
```

Comment:
Not sure if you need `cypress-merge` or `cypress-generate` in this package.json, these will not be called by CI/test-cypress.sh which calls `frontend/package.json`.  Here's a complicated diagram I created trying to show the process of how we generate a final `cypress-report.html` with results from ALL sub-packages:
![image](https://user-images.githubusercontent.com/12733153/114051697-53c81700-985b-11eb-9b10-f72c8e71d3b0.png)

Reply:
I thought it makes things easier, but as per above flow, It makes sense to remove these changes.